discord/commands/core.py

```diff
@@ -400,7 +398,7 @@ def reset_cooldown(self, ctx: ApplicationContext) -> None:
             The invocation context to reset the cooldown under.
         """
         if self._buckets.valid:
-            bucket = self._buckets.get_bucket(ctx)  # type: ignore # ctx instead of non-existent message
+            bucket = self._buckets.get_bucket(ctx)
```

Comment:
```suggestion
            bucket = await self._buckets.get_bucket(ctx)
```
`get_bucket` is async now. You forgot this in more places. Please check and correct all references.